src/parser/jobs/sam/modules/ReadyProcs.tsx

```diff
@@ -0,0 +1,120 @@
+import {Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {Event, Events} from 'event'
+import {filter, oneOf} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import Checklist, {Requirement, Rule} from 'parser/core/modules/Checklist'
+import {Procs as CoreProcs} from 'parser/core/modules/Procs'
+import React from 'react'
+import DISPLAY_ORDER from './DISPLAY_ORDER'
+
+export class ReadyProcs extends CoreProcs {
+	static override handle = 'ReadyProcs'
+	@dependency private checklist!: Checklist
+
+	theOtherOgi = 0
+	theOtherTendo = 0
+
+	TRACKED_ACTIONS = [
+		this.data.actions.KAESHI_NAMIKIRI.id,
+		this.data.actions.TENDO_GOKEN.id,
+		this.data.actions.TENDO_KAESHI_SETSUGEKKA.id,
+		this.data.actions.TENDO_KAESHI_GOKEN.id,
+		this.data.actions.TENDO_KAESHI_SETSUGEKKA.id,
+	]
+
+	override initialise() {
+		super.initialise()
+
+		const playerFilter = filter<Event>().source(this.parser.actor.id)
+		this.addEventHook(playerFilter.type('action').action(oneOf(this.TRACKED_ACTIONS)), this.onTrackedCast)
+	}
+
+	onTrackedCast(event: Events['action']): void {
+		if (event.action === this.data.actions.KAESHI_NAMIKIRI.id) {
+			this.theOtherOgi++
+		} else if (event.action === this.data.actions.TENDO_KAESHI_GOKEN.id || event.action === this.data.actions.TENDO_KAESHI_SETSUGEKKA.id) {
+			this.theOtherTendo++
+		}
+	}
+
+	override trackedProcs = [
+		{
+			procStatus: this.data.statuses.ZANSHIN_READY,
+			consumeActions: [this.data.actions.ZANSHIN],
+		},
+		{
+			procStatus: this.data.statuses.OGI_NAMIKIRI_READY,
+			consumeActions: [this.data.actions.OGI_NAMIKIRI],
+		},
+		{
+			procStatus: this.data.statuses.TENDO,
+			consumeActions: [
+				this.data.actions.TENDO_GOKEN,
+				this.data.actions.TENDO_SETSUGEKKA,
+			],
+		},
+		{
+			procStatus: this.data.statuses.TSUBAME_GAESHI_READY,
```

Comment:
Something is going on with this, might be a goof with the status data?
![image](https://github.com/xivanalysis/xivanalysis/assets/11688876/80cac34a-a447-4eb9-bdb9-63971a0cf28f)

Reply:
fixed typo.